src/integrationTest/java/uk/gov/hmcts/reform/iataskconfiguration/CamundaTaskInitiationTest.java

```diff
@@ -0,0 +1,293 @@
+package uk.gov.hmcts.reform.iataskconfiguration;
+
+import org.camunda.bpm.dmn.engine.DmnDecision;
+import org.camunda.bpm.dmn.engine.DmnDecisionRuleResult;
+import org.camunda.bpm.dmn.engine.DmnDecisionTableResult;
+import org.camunda.bpm.dmn.engine.DmnEngine;
+import org.camunda.bpm.dmn.engine.DmnEngineConfiguration;
+import org.camunda.bpm.engine.variable.VariableMap;
+import org.camunda.bpm.engine.variable.impl.VariableMapImpl;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Stream;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+@SuppressWarnings({"checkstyle:LineLength", "checkstyle:AbbreviationAsWordInName"})
+class CamundaTaskInitiationTest {
+
+    private DmnEngine dmnEngine;
+
+    @BeforeEach
+    void setUp() {
+        dmnEngine = DmnEngineConfiguration
+            .createDefaultDmnEngineConfiguration()
+            .buildEngine();
+    }
+
+    @DisplayName("Get task id")
+    @ParameterizedTest(name = "\"{0}\" \"{1}\" should go to \"{2}\"")
+    @CsvSource({
+        "submitAppeal, anything, processApplication, TCW, 2, false, ",
+        "submitTimeExtension, anything, decideOnTimeExtension, TCW, 2, true, Time extension",
+        "uploadHomeOfficeBundle, awaitingRespondentEvidence, reviewRespondentEvidence, TCW, 2, true, Case progression",
+        "submitCase, caseUnderReview, reviewAppealSkeletonArgument, TCW, 2, true, Case progression",
+        "submitReasonsForAppeal, reasonsForAppealSubmitted, reviewReasonsForAppeal, TCW, 2, true, Case progression",
+        "submitClarifyingQuestionAnswers, clarifyingQuestionsAnswersSubmitted, reviewClarifyingQuestionsAnswers, TCW, 2, true, Case progression",
+        "submitCmaRequirements, cmaRequirementsSubmitted, reviewCmaRequirements, TCW, 2, true, Case progression",
+        "listCma, cmaListed, attendCma, TCW, 2, true, Case progression",
+        "uploadHomeOfficeAppealResponse, respondentReview, reviewRespondentResponse, TCW, 2, true, Case progression",
+        "anything, prepareForHearing, createCaseSummary, TCW, 2, true, Case progression",
+        "anything, finalBundling, createHearingBundle, TCW, 2, true, Case progression",
+        "anything, preHearing, startDecisionsAndReasonsDocument, TCW, 2, true, Case progression",
+        "draftHearingRequirements, listing, reviewHearingRequirements, TCW, 2, true, Case progression, "
```

Comment:
Can we include a scenario for "Review the Appeal"?

Reply:
yes, I will add it.

Reply:
this is done here https://github.com/hmcts/ia-task-configuration/pull/44/files